src/common/crc32c_aarch64.c

```diff
@@ -147,7 +147,7 @@ uint32_t ceph_crc32c_aarch64(uint32_t crc, unsigned char const *buffer, unsigned
 			"mov    x16,            #0x8014         \n\t"
 			"movk   x16,            #0x8f15, lsl 16 \n\t"
 			"mov    v0.2d[0],       x16             \n\t"
-			:::"x16");
+			:::"x16","v0","v1");
```

Comment:
Oh, you mean `mov  v1.2d[0], x16` and `mov v0.2d[0], x16`, right?

Reply:
yes, v0 v1 have been changed here. the compiler may not know the changes.